### DISCUSSION TITLE
Is it possible to create a LeanTween.value equilavent?

### DISCUSSION THREAD

**Discussion by azbiusta** [KyryloKuzyk/PrimeTween#92] [General] [answered]
I was using LeanTween.value to change multiple things at once, like this:

```
void DoSomeAction() {
        LeanTween.value(gameObject, TweenChange, currentValue, targetValue, duration);
}
void TweenChange(float val)
{
        matEarth.SetFloat("_AlternateColorLerp", val);
        matHalo.SetFloat("_AlternateColorLerp", val);
        matLeaf.SetFloat("_AlternateColorLerp", val);
        matTree.SetFloat("_AlternateColorLerp", val);
        
        objectSpawner.UpdateTreeCount(val);
}

```

Is it possible to do something similar in PrimeTween, without creating tweens for each of the materials and the object spawner function?

**Comment by KyryloKuzyk** [comment, marked as answer]
Yes, of course! In PrimeTween this feature is called Tween.Custom(): https://github.com/KyryloKuzyk/PrimeTween?tab=readme-ov-file#custom-tweens
